metricbeat/module/dropwizard/_meta/config.reference.yml

```diff
@@ -2,6 +2,6 @@
   metricsets: ["collector"]
   period: 10s
   hosts: ["localhost:8080"]
-  metrics_path: /metrics/metrics
+  metrics_path: /test/metrics
```

Comment:
This is the reference configuration used in documentation, I would prefer not to show a "`test`" path here.

I see two alternatives:
* Look for another way of overriding this `metrics_path` in the integrations CI.
* Modify the path in the Dockerfile from `test` to `metrics` so this config works. Not sure if this path depends on the [name of the war](https://github.com/elastic/beats/blob/1edb43e594c80a18c061b04bccf54c4f972ff32b/metricbeat/module/dropwizard/_meta/Dockerfile#L13), or on the [artifact id](https://github.com/elastic/beats/blob/1edb43e594c80a18c061b04bccf54c4f972ff32b/metricbeat/module/dropwizard/_meta/test/pom.xml#L6). If this is changed, the config used by go and python tests would need to be updated.